marconi-chain-index/app/Main.hs

```diff
@@ -1,51 +1,30 @@
-{-# LANGUAGE OverloadedStrings   #-}
-{-# LANGUAGE ScopedTypeVariables #-}
-
+{-# LANGUAGE OverloadedStrings #-}
 module Main where
 
-import Control.Exception (catch)
-import Prettyprinter (defaultLayoutOptions, layoutPretty, pretty, (<+>))
-import Prettyprinter.Render.Text (renderStrict)
-
-import Cardano.Api qualified as C
-import Cardano.BM.Setup (withTrace)
-import Cardano.BM.Trace (logError)
-import Cardano.BM.Tracing (defaultConfigStdout)
-import Cardano.Streaming (ChainSyncEventException (NoIntersectionFound), withChainSyncEventStream)
 import Marconi.ChainIndex.CLI qualified as Cli
-import Marconi.ChainIndex.Indexers (filterIndexers, mkIndexerStream, startIndexers)
-import Marconi.ChainIndex.Logging (logging)
+import Marconi.ChainIndex.Indexers qualified as Indexers
 import System.Directory (createDirectoryIfMissing)
 
 main :: IO ()
 main = do
   o <- Cli.parseOptions
   createDirectoryIfMissing True (Cli.optionsDbPath o)
-  c <- defaultConfigStdout
-  withTrace c "marconi" $ \trace -> do
-    let indexers = filterIndexers (Cli.utxoDbPath o)
-                                  (Cli.addressDatumDbPath o)
-                                  (Cli.datumDbPath o)
-                                  (Cli.scriptTxDbPath o)
-                                  (Cli.epochStakepoolSizeDbPath o)
-                                  (Cli.mintBurnDbPath o)
-                                  (Cli.optionsTargetAddresses o)
-                                  (Cli.optionsNodeConfigPath o)
-    (cp, coordinator) <- startIndexers indexers
-    let preferredChainPoints =
-          -- If the user specifies the chain point then use that,
-          -- otherwise use what the indexers provide.
-          if Cli.optionsChainPoint o == C.ChainPointAtGenesis
-             then cp
-             else [Cli.optionsChainPoint o]
-    withChainSyncEventStream
-      (Cli.optionsSocketPath o)
-      (Cli.optionsNetworkId o)
-      preferredChainPoints
-      (mkIndexerStream coordinator . logging trace)
-      `catch` \NoIntersectionFound ->
-        logError trace $
-          renderStrict $
-            layoutPretty defaultLayoutOptions $
-              "No intersection found when looking for the chain point" <+> (pretty . Cli.optionsChainPoint) o <> "."
-                <+> "Please check the slot number and the block hash do belong to the chain"
+  let
+    maybeTargetAddresses = Cli.optionsTargetAddresses o
+    indexers =
```

Comment:
We don't filter indexers anymore? How can we disable indexers now?

Reply:
They are filtered within `runIndexers` depending on whether the second member of the tuple is a Nothing or Just ([here](https://github.com/input-output-hk/plutus-apps/pull/1014/files/368adb83addd41ae63053bf44be8a730579e6459#diff-79d117134d8ca4523adeb863332aa22e135e43fa0cf9be0f3765d56336347eeaR341))

Reply:
:+1:

Reply:
Ah no, you still have the `--disable-...` flags :D Forget it